src/tiled/preferences.cpp

```diff
@@ -566,6 +567,14 @@ void Preferences::setPluginEnabled(const QString &fileName, bool enabled)
     mSettings->setValue(QLatin1String("Plugins/Enabled"), enabledPlugins);
 }
 
+void Preferences::setZoomPref(bool mode){
+    mZoomPref = mode;
+}
+
+bool Preferences::getZoomPref(){
```

Comment:
Coding style:
* Getters do not have a "get" prefix, and should be marked `const`.
* The `{` for function bodies goes onto its own line.